api/tests/domain/demarches_simplifiees_test.py

```diff
@@ -179,137 +179,159 @@ def test_raise_error_in_unknown_state(self):
         assert error.value.errors == {"BankInformation": "Unknown Demarches Simplifiées state wrong"}
 
 
+VENUE_DMS_TOKEN_FIELD = {
+    "id": "Q2hhbXAtMjY3NDMyMQ==",
+    "label": "Identifiant du lieu",
+    "stringValue": "50a7536a21c8",
+    "value": "50a7536a21c8",
+}
+
+COMPANY_FIELD = {
+    "capitalSocial": "38130",
+    "codeEffectifEntreprise": "12",
+    "dateCreation": "2001-06-27",
+    "formeJuridique": "SAS, " "société " "par " "actions " "simplifiée",
+    "formeJuridiqueCode": "5710",
+    "inlineAdresse": "2 " "RUE " "LAMENNAIS, " "75008 " "PARIS " "8",
+    "nom": None,
+    "nomCommercial": "",
+    "numeroTvaIntracommunautaire": "FR67438391195",
+    "prenom": None,
+    "raisonSociale": "GAUMONT " "ALESIA",
+    "siren": "438391195",
+    "siretSiegeSocial": "43839119500056",
+}
+EXPECTED_RESULT = {
+    "status": "en_construction",
+    "updated_at": "2021-11-12T14:51:42+01:00",
+    "firstname": "John",
+    "lastname": "Doe",
+    "phone_number": "0102030405",
+    "siret": "43839119500056",
+    "siren": "438391195",
+    "iban": "FR7630001007941234567890185",
+    "bic": "QSDFGH8Z",
+    "annotation_id": "InterestingId",
+    "dossier_id": "Q2zzbXAtNzgyODAw",
+}
+DMS_TOKEN_FIELD_RESULT = {"dms_token": "50a7536a21c8"}
+
+
 class ParseRawBicDataTest:
     @pytest.mark.parametrize(
-        "entreprise",
+        "entreprise, identifiant_du_lieu, expected_result",
```

Comment:
(détail) on pourrait traduire les 2 premiers paramètres, non ? Suggestion : 

```suggestion
        "offerer_name, dms_token, expected_result",
```

Reply:
J'ai gardé l'idée de conserver le nom original du champ tel que renvoyé par DMS, puisque l'on paramétrise cela.
Si la francophonie te parait malvenue, on pourrait écrire:
``` suggestion
        "offerer_details_field, dms_token_field, expected_result",
```

Reply:
Non, tu as raison, laissons comme cela. Tu as bien fait.